src/amo/components/StaticPages/About/index.js

```diff
@@ -0,0 +1,140 @@
+              {
+                forumsLink: '<a href="https://discourse.mozilla-community.org/c/add-ons">forums</a>',
+              }), ['a'])}
+            />
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('This sounds great! How can I get involved?')}</h2>
+          <p>{i18n.gettext(`There are plenty of ways to get involved. If you're on the technical side:`)}</p>
+          <ul>
+            <li>
+              <Link to={'/developers'}>{i18n.gettext('Make your own add-on')}</Link>
+              {i18n.gettext('. We provide free hosting and update services and can help you reach a large audience of users.')}
+            </li>
+            <li
+              dangerouslySetInnerHTML={sanitizeHTML(i18n.sprintf(i18n.gettext(`If you have add-on development experience, 
+              %(start_link)sbecome an Add-on Reviewer%(end_link)s ! Our reviewers are add-on fans 
+              with a technical background who review add-ons for code quality and stability.`),
+              {
+                start_link: '<a href="https://wiki.mozilla.org/Add-ons/Reviewers/Applying">',
+                end_link: '</a>',
+              }), ['a'])}
+            />
+            <li
+              dangerouslySetInnerHTML={sanitizeHTML(i18n.sprintf(i18n.gettext(`Help improve this website.
+              It's open source, and you can file bugs and submit patches.
+              %(github)s contains all of our current bugs, legacy bugs can still be found in Bugzilla.`),
+              {
+                github: '<a href="https://github.com/mozilla/olympia/issues">GitHub</a>',
+              }), ['a'])}
+            />
+          </ul>
+          <p>{i18n.gettext(`If you're interested in add-ons but not quite as technical, there are still ways to help:`)}</p>
+          <ul>
+            <li>{i18n.gettext('Tell your friends! Let people know which add-ons you use.')}</li>
+            <li
+              dangerouslySetInnerHTML={sanitizeHTML(i18n.sprintf(i18n.gettext(`Participate in our %(forumsLink)s.`),
+                {
+                  forumsLink: '<a href="https://discourse.mozilla-community.org/c/add-ons">forums</a>',
+                }), ['a'])
+              }
+            />
+            <li>
+              {i18n.gettext('Review add-ons on the site. Add-on authors are more likely to improve their add-ons and write new ones when they know people appreciate their work.')}
+            </li>
+          </ul>
+        </section>
+        <section>
+          <h2>{i18n.gettext('I have a question')}</h2>
+          <p>{i18n.gettext('A good place to start is our')}&nbsp;
+            <Link to="/faq"><abbr title="{i18n.gettext('Frequently Asked Questions')}">FAQ</abbr></Link>
+            {i18n.gettext(`. If you don't find an answer there, you can`)}&nbsp;<a href="https://discourse.mozilla-community.org/c/add-ons">{i18n.gettext('ask on our forums')}</a>.
```

Comment:
This should make the link part of the translation as done elsewhere otherwise the context is split up and that might make it hard to localise.

Reply:
The non-breaking space should be removed too.

Reply:
`i18n.gettext` trims the start and end spaces, so I need to put `&nbsp;`